src/xdocs/config_naming.xml

```diff
@@ -369,11 +369,30 @@
           An example of how to configure the check for names that begin with
           a lower case letter, followed by any letters or digits is:
         </p>
+        <p>Configuration:</p>
         <source>
 &lt;module name="CatchParameterName"&gt;
   &lt;property name="format" value="^[a-z][a-zA-Z0-9]+$"/&gt;
 &lt;/module&gt;
         </source>
+        <p>Code example:</p>
+        <pre>
+public class TestCatchParameterNameCheck {
+  public void MyTest() {
+    try {
+      // ...
+    } catch (ArithmeticException ex) { //OK
+      // ...
+    } catch (ArrayIndexOutOfBoundsException ex2) { //OK
+      // ...
+    } catch (IOException thirdException) { //OK
+      // ...
+    } catch (Exception FourthException) { //violation, the initial letter should be lowercase
```

Comment:
"violation, the first letter should be in lowercase".

@rnveach , please suggest better wording if required.

Reply:
minor change `should be lowercase`